test/integration/functional_test_tunnel_test.go

```diff
@@ -131,6 +133,49 @@ func validateTunnelStart(ctx context.Context, t *testing.T, profile string) {
 	tunnelSession = *ss
 }
 
+// validateNoSecondTunnel ensures more than 1 tunnel can NOT run simultaneously
+func validateNoSecondTunnel(ctx context.Context, t *testing.T, profile string) {
```

Comment:
these tests all run in parallel, so ("StartTunnel", validateTunnelStart) test might be running before this and would cause niether of them be running,..


so I recommend adding this test Under Same test "validateTunnelStart" and right after Starting tunnel, we can run it one more time, just to ensure second time wont run.

Reply:
@medyagh according to my understanding the tests within this file, based on code [here](https://github.com/kubernetes/minikube/blob/ba699acc9bf090d466453d19c6f8caaeb8992ff5/test/integration/functional_test_tunnel_test.go#L59) and [here](https://github.com/kubernetes/minikube/blob/ba699acc9bf090d466453d19c6f8caaeb8992ff5/test/integration/functional_test_tunnel_test.go#L72), will run serially, in order according to the list defined [here](https://github.com/kubernetes/minikube/blob/ba699acc9bf090d466453d19c6f8caaeb8992ff5/test/integration/functional_test_tunnel_test.go#L64). I've put this new test before `validateTunnelStart` to ensure that we don't face the scenario you mentioned. Please correct me if I'm wrong! If you still think I should put it inside `validateTunnelStart`, let me know, I'm happy to make the change.